src/util/debug_module.js

```diff
@@ -21,6 +21,7 @@ const path = require('path');
 const util = require('util');
 const os = require('os');
 const debug_config = require('./debug_config');
+const env_config = require('../../config.js');
```

Comment:
Nitpick: We usually call it just `config`. So `const env_config = require('../../config.js')`.